### ISSUE TITLE
Indicate "fisher" installation

### ISSUE THREAD

**Issue by metal3d** [simnalamburt/shellder#15]
Not very important but I'm using fisher, I did:

```
fisher simnalamburt/shellder
```

Installation is ok. So maybe you may write this in the README file ;)

**Comment by ghost** [comment]
`fisher i shellder` or even `fisher shellder` (although I don't like that notation) are enough.

**Comment by tommyjcarpenter** [comment]
```
~ fisher install shellder                                                                                 Wed Aug 17 21:20:41 2016
Installing 1 plugin/s
! Fetch shellder github.com/fisherman/shellder
! There was an error installing shellder or more plugin/s.
Try using a namespace before the plugin name: owner/shellder
~ fisher install simnalamburt/shellder                                                           1176ms  Wed Aug 17 21:20:46 2016
Installing 1 plugin/s
OK Fetch shellder github.com/simnalamburt/shellder
Done in 1s 225ms
 ~ 
```

**Comment by simnalamburt** [comment]
Fixed with 7227d4282a63bf308be827ec796244fbecbbd4d7
